docs/interchain-staking.md

```diff
@@ -0,0 +1,178 @@
+# Interchain Staking
+
+
+# Introduction
+
+Interchain Staking has been referred to in many different ways: Shared Security, Cross Chain Validation, Cross Chain Collateralization, Shared Staking.  This document will restrict use to the following three terms:
+
+
+
+*   **Shared Security**
+    *   Shared security is a family of technologies that include optimistic rollups, zk-rollups, sharding and interchain staking.
+*   **Interchain Staking**
+    *   Interchain Staking is the Cosmos specific category of Shared Security that uses IBC (Inter-Blockchain Communication).
+*   **Cross Chain Validation**
+    *   Cross Chain Validation is the specific IBC level protocol that enables Interchain Staking.
+
+While there are many ways that Interchain Staking could take place, this document will focus on one instance of Interchain Staking that has particularly valuable attributes for the ATOM token and the Cosmos Hub. The resulting technology may be applied to other scenarios with little to no modification, but I will leave those out for now (or only dedicate a small section) since the current priority is to implement this feature for the Cosmos Hub.
+
+At a very high level, Interchain Staking is the ability for staking tokens that have been delegated to validators on a parent chain to inform the composition of a validator set on a child chain. Inter-Blockchain Communication is utilized to relay updates of validator stake delegations from the parent chain to the child chain so that the child chain will have an up-to-date model of which validators can produce blocks on the child chain. The inclusion of parent chain validators is opt-in. The parent chain will honor any proof of validator misbehaviour produced by the child chain as evidence that results in slashing the stake of misbehaving validators on the parent chain. In this way the security gained from the value of the stake locked on the parent chain will be shared with the child chain.
```

Comment:
```suggestion
At a very high level, Interchain Staking is the ability for staking tokens that have been delegated to validators on a parent chain to inform the composition of a validator set on a child chain. Inter-Blockchain Communication is utilized to relay updates of validator stake delegations from the parent chain to the child chain so that the child chain will have an up-to-date model of which validators can produce blocks on the child chain. The inclusion of parent chain validators can be mandatory or opt-in depending on the requirements of the child chain. The parent chain will honor any proof of validator misbehavior produced by the child chain as evidence that results in slashing the stake of misbehaving validators on the parent chain. In this way the security gained from the value of the stake locked on the parent chain will be shared with the child chain.
```